apis/kueue/v1beta1/workload_types.go

```diff
@@ -59,6 +59,9 @@ type WorkloadSpec struct {
 	// +kubebuilder:default=""
 	// +kubebuilder:validation:Enum=kueue.x-k8s.io/workloadpriorityclass;scheduling.k8s.io/priorityclass;""
 	PriorityClassSource string `json:"priorityClassSource,omitempty"`
+
+	// Queueing that will determine if a job needs to be requeued or not after being suspended
```

Comment:
We should enumerate the supported policies in the API docs, see https://github.com/kubernetes/api/blob/master/batch/v1/types.go#L214-L228 as one example